packages/react/src/components/ContainedList/ContainedList.tsx

```diff
@@ -50,16 +60,16 @@ function renderChildren(children) {
   return children;
 }
 
-function ContainedList({
+const ContainedList: React.FC<ContainedListProps> = ({
   action,
   children,
   className,
-  isInset,
-  kind = variants[0],
+  isInset = false,
```

Comment:
Why does this need a default now?

Reply:
I had added the default so that isInset is not undefined anywhere.l